src/TableAutocomplete/Item.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export interface TableAutocompleteItemProps {
+  /**
+   * String representation of the option.
+   */
+  value: string;
+  /**
+   * Custom typeahead string. By default, typeahead uses `value`.
+   */
+  searchValue?: string;
+  children?: React.ReactNode;
+}
+
+const TableAutocompleteItem: React.FC<TableAutocompleteItemProps> = ({
+  children,
+  value,
+}) => <>{children || value}</>;
```

Comment:
This component mainly exists to collect props that will be used by the parent, TableAutocomplete.

This "prop collector" pattern is used throughout NDS to provide cleaner subcomponent interfaces for developers